fleet-packages-quickstart/README.md

```diff
@@ -0,0 +1,246 @@
+
+
+Create a cluster in us-west:
+```
+gcloud container clusters create fp-quickstart-cluster-west --project=$PROJECT_ID --zone=us-west2-a --workload-pool=$PROJECT_ID.svc.id.goog --enable-fleet --async
+```
+
+Create a cluster in us-east:
+```
+gcloud container clusters create fp-quickstart-cluster-east --project=$PROJECT_ID --zone=us-east1-c --workload-pool=$PROJECT_ID.svc.id.goog --enable-fleet --async
+```
+
+It will take a few minutes for the clusters to be ready. You can continue with the next steps and there is a later step to verify that they are ready.
+
+
+## Configure Cloud Build Repositories
+
+We have a sample package at https://github.com/GoogleCloudPlatform/anthos-config-management-samples/tree/main/fleet-packages-quickstart/config. This is a simple example that just sets up nginx. Fork this repo into your own GitHub account.
+
+Fleet Packages leverages [Cloud Build Repositories] (2nd gen) for connecting to git. Follow these steps to set it up:
+
+1. Enable the Cloud Build and Secret Manager APIs:
+```
+gcloud services enable cloudbuild.googleapis.com --project=$PROJECT_ID
+gcloud services enable secretmanager.googleapis.com --project=$PROJECT_ID
+```
+
+2. Verify the IAM permissions as described in https://cloud.google.com/build/docs/automating-builds/github/connect-repo-github?generation=2nd-gen#iam_perms.
+
+3. Create a connection to your GitHub host using the steps described in https://cloud.google.com/build/docs/automating-builds/github/connect-repo-github?generation=2nd-gen#connecting_a_github_host. You can use either the console or gcloud. You must use the region `us-central1` and we suggest using the name `fp-quickstart` for the connection (the rest of the guide will use this name).
+
+4. Connect to the forked GitHub repository using the steps described in https://cloud.google.com/build/docs/automating-builds/github/connect-repo-github?generation=2nd-gen#connecting_a_github_repository_2. You can use either the console or gcloud. Again, you must use the region `us-central1`, connection name `fp-quickstart`, and we suggest using the name `anthos-config-management-samples` for the repository.
+
+5. We will need the fully qualified name for the repository a little later. With the suggested names, it will be:
+
+```
+projects/$PROJECT_ID/locations/us-central1/connections/fp-quickstart/repositories/anthos-config-management-samples
+```
+
+If you are uncertain about then names, you can list these with gcloud using the following commands:
+
+List all connections:
+
+```
+gcloud builds connections list --region=us-central1 --project=$PROJECT_ID
+```
+
+List all repositories for a specific connection:
+
+```
+gcloud builds repositories list --region=us-central1 --connection=<CONNECTION_NAME>
+```
+
+## Configure a Service Account
+
+Create a service account:
+
+```
+gcloud iam service-accounts create fp-quickstart-sa
+```
+
+Add an IAM policy binding for the ResourceBundle Publisher role:
```

Comment:
Link to https://cloud.google.com/iam/docs/understanding-roles#configdelivery.resourceBundlePublisher as well

Reply:
Done